CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.76.0 (2024-02-15)
```

Comment:
You'll also need to update `package.json` and `package-lock.json` with this new version number in order for the library to be released.

Reply:
Updated